### PR TITLE
Deprecate `SECURE_*` keycodes for `QK_SECURE_*`

### PR DESCRIPTION
**Pull request by zvecr**
## Description

Also adds short form aliases.

## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/feature_secure.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ SECURE_ENABLE = yes
...
-| `SECURE_LOCK`    | Revert back to a locked state                                                  |
-| `SECURE_UNLOCK`  | Forces unlock without performing a unlock sequence                             |
-| `SECURE_TOGGLE`  | Toggle directly between locked and unlock without performing a unlock sequence |
-| `SECURE_REQUEST` | Request that user perform the unlock sequence                                  |
+| Key                 |Aliases  | Description                                                                    |
+|---------------------|---------|--------------------------------------------------------------------------------|
+| `QK_SECURE_LOCK`    |`SE_LOCK`| Revert back to a locked state                                                  |
+| `QK_SECURE_UNLOCK`  |`SE_ULCK`| Forces unlock without performing a unlock sequence                             |
+| `QK_SECURE_TOGGLE`  |`SE_TOGG`| Toggle directly between locked and unlock without performing a unlock sequence |
+| `QK_SECURE_REQUEST` |`SE_REQ` | Request that user perform the unlock sequence                                  |
```
Comment (zvecr) [inline docs/feature_secure.md:34]: `SE_RQST` maybe?
```suggestion
| `QK_SECURE_REQUEST` |`SE_RQST`| Request that user perform the unlock sequence                                  |
```

Comment (fauxpark) [reply docs/feature_secure.md:34]: `REQ` is fine, doesn't always have to be 4 chars.

File: docs/feature_secure.md [hunk 2]
```diff
@@ -26,12 +26,12 @@ SECURE_ENABLE = yes
...
-| Key              | Description                                                                    |
-|------------------|--------------------------------------------------------------------------------|
-| `SECURE_LOCK`    | Revert back to a locked state                                                  |
-| `SECURE_UNLOCK`  | Forces unlock without performing a unlock sequence                             |
-| `SECURE_TOGGLE`  | Toggle directly between locked and unlock without performing a unlock sequence |
-| `SECURE_REQUEST` | Request that user perform the unlock sequence                                  |
+| Key                 |Aliases  | Description                                                                    |
+|---------------------|---------|--------------------------------------------------------------------------------|
+| `QK_SECURE_LOCK`    |`SE_LOCK`| Revert back to a locked state                                                  |
+| `QK_SECURE_UNLOCK`  |`SE_ULCK`| Forces unlock without performing a unlock sequence                             |
```
Comment (fauxpark) [inline docs/feature_secure.md:32]:
```suggestion
| `QK_SECURE_UNLOCK`  |`SE_UNLK`| Forces unlock without performing a unlock sequence                             |
```

File: quantum/quantum_keycodes.h [hunk 3]
```diff
@@ -857,6 +857,12 @@ enum quantum_keycodes {
 #define KO_ON QK_KEY_OVERRIDE_ON
 #define KO_OFF QK_KEY_OVERRIDE_OFF
 
+// Secure
+#define SE_LOCK QK_SECURE_LOCK
+#define SE_ULCK QK_SECURE_UNLOCK
```
Comment (fauxpark) [inline quantum/quantum_keycodes.h:862]:
```suggestion
#define SE_UNLK QK_SECURE_UNLOCK
```

**Review by fauxpark — APPROVED** [review]

**Pull request merged**